### DISCUSSION TITLE
How to set size of Gaster Blasters

### DISCUSSION THREAD

**Discussion by Hoverworm** [Jcw87/c2-sans-fight#125] [Q&A] [answered]
Hello. I can't figure out how to set the size of the Gaster Blaster in the code when making custom attacks. I tried 0.1 and that makes it very slim, 5 makes it tiny. I even put negatives in, but it didn't seem to change anything. I checked the documentation, and it doesn't mention the "Size" argument anywhere else or explain what it is. Could someone explain the argument to me?

**Comment by Jcw87** [comment, marked as answer]
There are only 3 supported sizes.
0 - The thin gasterblaster.
1 - The properly scaled gaster blaster.
2 - The bigger properly scaled gasterblaster.
If you use any other number, it will use a default that was never intended for use.

**Reply by Hoverworm** [reply]
Ah, okay. I was thinking that it would take a numeric value and set its size accordingly. Thank you.
